types/test/tsconfig.json

```diff
@@ -3,7 +3,9 @@
 		"target": "es5",
 		"module": "es2015",
 		"strict": true,
-		"noEmit": true
+		"noEmit": true,
+		"skipLibCheck": true,
+		"moduleResolution": "node"
```

Comment:
I think `moduleResolution: 'node'` is only required against the latest v2.9 chart.js type definitions (because of [this change](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41710)). Please check against v3 if it's actually required.

Reply:
Still required, fails without it.

Reply:
Ok, not a big deal. Can you revert that change though, it was already defined at line 5.

Reply:
I still want to revert that line to the previous location.